apps/mobile/package.json

```diff
@@ -142,7 +143,7 @@
     "babel-plugin-macros": "3.1.0",
     "babel-preset-expo": "11.0.6",
     "concurrently": "8.2.2",
-    "eslint": "8.53.0",
+    "eslint": "9.14.0",
```

Comment:
unintended I imagine

Reply:
I saw some warnings about it being out of date so updated it. I should have removed from here